### ISSUE TITLE
Is it possible to create a `Context` to use in test

### ISSUE THREAD

**Issue by sindreij** [async-graphql/async-graphql#775]
Hi. Thank you for a really nice library, it's been a dream to use.

I'm looking into creating some (unit-)tests for the graphql api, and since the Object methods are just plain methods, it would be nice if I were able to call them directly. However as far as I can tell there is no documented way of creating a `Context`. Is this something that is possible?

**Comment by Miaxos** [comment]
Yes you can with this: `async_graphql::context::QueryEnv::create_context` these methods & structs are hidden but you can use it inside your tests.

**Comment by sindreij** [comment]
Thanks

**Comment by sindreij** [comment]
Thanks!

**Comment by sindreij** [comment]
I cannot figure out how to create a `SchemaEnv`, which is required by `create_context`.

**Comment by dspicher** [comment]
I am having the same issue.

**Comment by sindreij** [comment]
@Miaxos: Is it possible to create a `SchemaEnv`, required by `create_context`? Or have I misunderstood how to use `create_context`?

**Comment by sadokmtir** [comment]
@sindreij  did you find any solution so far ?

**Comment by sindreij** [comment]
Nope, unfortunately
